packages/docregistry/src/context.ts

```diff
@@ -496,13 +425,109 @@ class Context<T extends DocumentRegistry.IModel> implements DocumentRegistry.ICo
         name,
         language: this._model.defaultKernelLanguage,
       };
-      return this.session.initialize();
+      this.session.initialize();
```

Comment:
I wonder if we shouldn't even wait on a checkpoint to resolve the ready promise. Thoughts?

Reply:
Works for me, updated.